service/src/main/java/uk/gov/hmcts/reform/fpl/model/ApplicantParty.java

```diff
@@ -8,26 +8,46 @@
 import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
 import uk.gov.hmcts.reform.fpl.model.common.Party;
 import uk.gov.hmcts.reform.fpl.model.common.Telephone;
+import uk.gov.hmcts.reform.fpl.model.interfaces.TelephoneContacts;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasContactDirection;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasTelephoneOrMobile;
 
 import java.util.Date;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+
 @Data
 @EqualsAndHashCode(callSuper = true)
+@HasTelephoneOrMobile
+@HasContactDirection
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class ApplicantParty extends Party {
+public class ApplicantParty extends Party implements TelephoneContacts {
+    @NotBlank(message = "Enter the applicant's full name")
     private final String organisationName;
     private final Telephone mobileNumber;
+    @NotBlank(message = "Enter a job title for the contact")
     private final String jobTitle;
     private final String pbaNumber;
 
+    public Telephone getTelephoneNumber() {
```

Comment:
Why you needed this? These should be generated for us by Lombok.

Reply:
At one point builder was complaining I was not implementing interface correctly. Seems to be quiet now. Removing